### PR TITLE
Adding capability to keep dorking across user news feed

### PR DESCRIPTION
**Pull request by ffr4nz**
Use a Github account to monitor new code from users or projects. Just keep dorking over new code on repositories that you are watching. Useful to monit your company repositories.

### CODE REVIEW

**Comment by techgaun** [comment]
Hi @ffr4nz thanks for the PR. It looks good but I want to check it by running once. I appreciate your contribution.

**Comment by ffr4nz** [comment]
@techgaun no problem. Test it and if you have any feedback just let me know!

**Comment by techgaun** [comment]
@ffr4nz I tried quickly and saw an issue of mixed tabs and spaces in indentation. Could you fix it to use 4-space soft tabs. Below is the log:

```
python github-dork.py -u dev-nepal
  File "github-dork.py", line 41
    search(
          ^
TabError: inconsistent use of tabs and spaces in indentation
```

File: github-dork.py [hunk 1]
```diff
@@ -35,12 +36,43 @@ def search_wrapper(gen):
...
+    else:
+	monit(  gh_dorks_file,
+		active_monit
+	)	
+
+def monit(gh_dorks_file=None,active_monit=None):
+    if gh_user is None:
+	raise Exception('Error, env Github user variable needed')
+    else:
+	print "Monitoring user private feed searching new code to be 'dorked'. Every new merged pull request trigger user scan."
```
Comment (techgaun) [inline github-dork.py]: @ffr4nz Could you put these in a brace such as `print("Monitoring...")`? Its in bunch of places. This ensures it does not break on python 3

**Comment by ffr4nz** [comment]
Hi @techgaun. Changes have been made... check it when you can. Thanks

**Comment by techgaun** [comment]
thanks @ffr4nz looks good now

**Pull request merged**